kaltura_process_cue_points.rb

```diff
@@ -4,93 +4,92 @@
 include Kaltura
 
 def process_slides(xml)
-  slides_array=[]
+  slides_array = []
   xml.xpath('//section').each do |section|
-    #position=section.at_xpath('@position')
-    content=section.at_xpath('content').text
-    title=section.at_xpath('title').text
-    my_slide={:title=>title,:content=>content}
+    # position=section.at_xpath('@position')
+    content = section.at_xpath('content').text
+    title = section.at_xpath('title').text
+    my_slide = { title: title, content: content }
     slides_array.push(my_slide)
   end
-  return slides_array
+  slides_array
 end
 
-def ingest_to_kaltura(client, base_endpoint, partner_id, secret, parent_cat_id, full_cat_path, cat_name, entry_name, meeting_id, vid_file_path)
-
+def ingest_to_kaltura(client, _base_endpoint, _partner_id, _secret, parent_cat_id, full_cat_path, cat_name, entry_name, meeting_id, vid_file_path)
   # check whether category already exists
-  filter = KalturaCategoryFilter.new()
-  filter.full_name_equal = full_cat_path + ">" +cat_name
-  pager = KalturaFilterPager.new()
+  filter = KalturaCategoryFilter.new
+  filter.full_name_equal = full_cat_path + '>' + cat_name
+  pager = KalturaFilterPager.new
   results = client.category_service.list(filter, pager)
   # if not, create it
-  if !results.total_count
-	      category = KalturaCategory.new()
-	      category.parent_id=parent_cat_id
-	      category.name = cat_name
-	      results = client.category_service.add(category)
-	      puts("Created category: " + cat_name + ", cat ID: "+results.id)
+  unless results.total_count
+    category = KalturaCategory.new
+    category.parent_id = parent_cat_id
+    category.name = cat_name
+    results = client.category_service.add(category)
+    puts('Created category: ' + cat_name + ', cat ID: ' + results.id)
   end
-  upload_token = KalturaUploadToken.new()
+  upload_token = KalturaUploadToken.new
```

Comment:
This is again a style issue [both forms will work equally well] but I am sorry, I prefer my original line.

Reply:
I would actually agree, VSCode linter did this.